prow/jobs/test-infra/tekton-demo.yaml

```diff
@@ -0,0 +1,146 @@
+presubmits:
+  kyma-project/test-infra:
+    - name: tekton-demo
```

Comment:
Please remove this file, or move it as an example to the docs. If it's a demo job this should not be added to the Prow.